client/app/veeam-enterprise/veeam-enterprise.service.js

```diff
@@ -0,0 +1,58 @@
+(() => {
+    class VeeamEnterpriseService {
+        constructor ($q, $translate, OvhApiVeeamEnterprise) {
+            this.$q = $q;
+            this.$translate = $translate;
+            this.veeamEnterprise = OvhApiVeeamEnterprise.v6();
+
+            this.unitOfWork = { };
```

Comment:
How do we know `init` is a function if you don't name it as a verb ? It could be the short for `initialValue`

```js
this.unitOfWork = {
	initialize: () => (this.unitOfWork.messages = [])
}
```